assets/src/components/map/tooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { Tooltip } from "react-leaflet"
+
+interface MapTooltipProps {
+  title: string
+  permanent?: boolean
+}
+
+export const MapTooltip = ({ title, permanent }: MapTooltipProps) => {
+  return (
```

Comment:
nit(non-blocking): the braces and `return` keyword could be removed

Reply:
https://github.com/mbta/skate/pull/2366/commits/22f35ce287a76ed7db9662696bf1e56cda339011